### DISCUSSION TITLE
q-timeline and v-for

### DISCUSSION THREAD

**Discussion by alessioguidi** [quasarframework/quasar#12191] [General - Components / Directives / etc]
I don't understand why the vertical line between one element and another is not displayed using the v-for directive
See example
https://codepen.io/alessioguidi/pen/xxPxPym?editors=1010

Thanks

**Comment by MilosPaunovic** [comment]
You have to loop through the `QTimelineEntry` elements:
[https://codepen.io/milospaunovic/pen/Rwjwxwr?editors=1010](https://codepen.io/milospaunovic/pen/Rwjwxwr?editors=1010)

**Comment by alessioguidi** [comment]
Hi Milos,
Sorry but my example was incomplete.
I need to iterate over a parent div like this
https://codepen.io/alessioguidi/pen/xxPxPym?editors=1010

All other components work regularly in this way, this one does not.
Is there a reason?

Many thanks

Il giorno mar 25 gen 2022 alle ore 19:51 Milos Paunovic <
***@***.***> ha scritto:

**Reply by MilosPaunovic** [reply]
If you iterate over `QTimeline`, you're gonna create separate elements which can't be connected with lines. If you iterate over `QTimelineEntry` elements, then you'll have connecting lines.

**Comment by alessioguidi** [comment]
Yes, of course, but now I have a div in the middle
see new codepen https://codepen.io/alessioguidi/pen/xxPxPym?editors=1010

Thanks


Il giorno mar 25 gen 2022 alle ore 20:08 Milos Paunovic <
***@***.***> ha scritto:

**Reply by pdanpdan** [reply]
https://codepen.io/pdanpdan/pen/YzEzeaL?editors=1010

**Comment by MilosPaunovic** [comment]
If you want the connecting lines, `QTimelineEntry` elements must be direct descendants of `QTimeline`.

**Comment by alessioguidi** [comment]
Ok, thanks

Il giorno mar 25 gen 2022 alle ore 20:12 Milos Paunovic <
***@***.***> ha scritto:
